content/applications/hr/recruitment/schedule_interviews.rst

```diff
@@ -0,0 +1,329 @@
+    enter the date the meeting repeats. If :guilabel:`Day of month` is selected, use the drop-down
+    menus to determine the frequency. Select either :guilabel:`First`, :guilabel:`Second`,
+    :guilabel:`Third`, :guilabel:`Fourth`, or :guilabel:`Last` for the first drop-down menu, then
+    select the day (:guilabel:`Monday`, :guilabel:`Tuesday`, etc.) in the second drop-down menu.
+  - :guilabel:`Until`: using the drop-down menu, select when the meetings stop repeating. The
+    available options are :guilabel:`Number of repetitions`, :guilabel:`End date`, and
+    :guilabel:`Forever`. If :guilabel:`Number of repetitions` is selected, enter the number of
+    total meetings to occur in the blank field. If :guilabel:`End date` is selected, specify the
+    date using the calendar pop-up window, or type in a date in a XX/XX/XXXX format.
+    :guilabel:`Forever` schedules meetings indefinitely.
+
+- :guilabel:`Location`: enter the location for the meeting.
+- :guilabel:`Tags`: select any tag(s) for the meeting using the drop-down menu. There is no limit to
+  the number of tags that can be used.
+- :guilabel:`Appointment`: if an appointment is associated with this meeting, select it form the
+  drop-down menu, or create a new appointment by typing in the appointment name, then click
+  :guilabel:`Create & Edit...`. A :guilabel:`Create Appointment` form loads. Enter the information
+  on the form, then click :guilabel:`Save & Close`.
+- :guilabel:`Privacy`: select if the organizer appears either :guilabel:`Available` or
+  :guilabel:`Busy` for the duration of the meeting, using the drop-down menu. Next, select the
+  visibility of this meeting, using the drop-down menu to the right of the first selection. Options
+  are :guilabel:`Public`, :guilabel:`Private`, and :guilabel:`Only internal users`.
+  :guilabel:`Public` allows for everyone to see the meeting, :guilabel:`Private` allows only the
+  attendees listed on the meeting to see the meeting, and :guilabel:`Only internal users` allows
+  anyone logged into the company database to see the meeting.
+- :guilabel:`Organizer`: the employee who created the meeting is populated in this field. Use the
+  drop-down menu to change the selected employee.
+- :guilabel:`Reminders`: select a reminder from the drop-down menu. Default options include
+  :guilabel:`Notification`, :guilabel:`Email`, and :guilabel:`SMS Text Message`, each with a
+  specific time period before the event (hours, days, etc). The reminder chosen alerts the meeting
+  participants of the meeting via the selected option at the specified time. Multiple reminders can
+  be selected in this field.
+
+.. image:: schedule_interviews/new-event.png
+   :align: center
+   :alt: A new meeting card with all the details populated, and ready to save.
+
+Send meeting to attendees
+-------------------------
+
+Once changes have been entered and the meeting details are correct, the meeting can be sent to the
```

Comment:
Which form should I be looking at right now? (: I think the context would be helpful for readers skimming through the doc!